openpype/pipeline/workfile/build_workfile.py

```diff
@@ -102,11 +100,17 @@ def build_workfile(self):
             List[Dict[str, Any]]: Loaded containers during build.
         """
 
+        from openpype.pipeline.context_tools import (
+            get_current_project_name,
+            get_current_asset_name,
+            get_current_task_name,
+        )
```

Comment:
Any reason why these imports can't be at the top of the file?

Reply:
Causing cross import bug.